cmd/commands/import.go

```diff
@@ -31,6 +31,7 @@ func NewImportCommand(c *Context) *cobra.Command {
 				passCB = func() ([]byte, error) { return []byte(password), nil }
 			} else {
 				passCB = func() ([]byte, error) {
+					fmt.Println()
 					fmt.Print("Enter Password: ")
```

Comment:
Can this be refactored into one line ?
```
fmt.Println("Enter Password: ")
```

Reply:
Password is applicable only for encrypted keys. So this prompt appears only for encrypted keys, not required for un-encrypted keys.